libraries/core/classes/BrightnessRequest/index.js

```diff
@@ -0,0 +1,80 @@
+import event from '../Event';
+import AppCommand from '../AppCommand';
+import { logger } from '../../helpers';
+
+const LIB_VERSION = '17.0';
+const GET_COMMAND_NAME = 'getCurrentBrightness';
+const RESPONSE_EVENT_NAME = 'currentBrightnessResponse';
+
+/**
+ * Brightness request handler.
+ *
+ * Maintains the request command and waits for a response to resolve a promise returned by
+ * `.dispatch`.
+ */
+export class BrightnessRequest {
+  /**
+   * Creates a get command instance.
+   * @returns {AppCommand}
+   */
+  static createGetCommand() {
```

Comment:
It is unnecessary to have this as a static method there. It is a bit weird to access it via `this.constructor`. Just strip it out as a separate function here in the file: `const createGetCommand = () => { ... }`. Best place: right before the class definition.

Reply:
I've moved the command creation into the `dispatch` method, since it only happens there. didn't make so much sense to have it within a separate method.